DMIS.md

```diff
@@ -61,7 +61,7 @@ As of Feb 2022, I'm not sure if (1) is still true:
 
 ```
 select docid_length,count(*) from(
-select length(to_char(docid)) as docid_length from APSD.t_ssb_trip_current@garfo_nefsc where docid is not null) group by docid_length;
+select length(to_char(docid)) as docid_length from NEFSC_GARFO.APSD_t_ssb_trip_current@garfo_nefsc where docid is not null) group by docid_length;
```

Comment:
again the @garfo_nefsc tag is not in the name of the table on NEFSC_GARFO

Reply:
0cb6924 and 0cb6924